### DISCUSSION TITLE
Outlook Email Limit for Ghosts Clients

### DISCUSSION THREAD

**Discussion by Timmythetim** [cmu-sei/GHOSTS#172] [Q&A]
I was working on getting Outlook working on my GHOSTS clients. I ran into an issue with the my clients getting understandably flagged for suspicious activity as I tried to send emails. I am wondering if anyone has any advice when it comes to this?

**Comment by sei-dupdyke** [comment]
Hi - hmm, do you mean flagged by antivirus? Or something else?

**Reply by Timmythetim** [reply]
It seems to be something else. We have our clients sending emails relatively quickly to collect data. After a few attempts, we receive what seems to be an automated email reporting that the account has been detected with unusual activity. It seems to lock down the account after roughly 10 emails.

**Comment by sei-dupdyke** [comment]
Hmm, what mail server? Do you control it by chance?

**Reply by vlombardi22** [reply]
I am running into the same problems as the original author actually. I was just using standard outlook accounts. Do you guys use your own server?

**Reply by sei-dupdyke** [reply]
Yes, we run an air-gapped range, controlling everything from top to bottom.

One of the constant pains of automating email is that so many bad guys also want to do this, so it's tough to do it legitimately unless we control everything.

**Reply by vlombardi22** [reply]
I figured as much but wanted to check if there was a simpler way to do it.

**Reply by sei-dupdyke** [reply]
No, sorry.
